src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -216,17 +221,17 @@
     <RuntimeGroup Include="rhel">
       <Parent>linux</Parent>
       <Architectures>x64</Architectures>
-      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6</Versions>
+      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6;7.7;7.8;7.9;7.10</Versions>
```

Comment:
Same about 7.10 here.

Reply:
since the `ol` did not have any minor versions, we can keep doing that as well, right? e.g. just add `ol 9`?

Reply:
Why would we have to fix everything? We could simply say that the RIDs for `ol` don't include minor version in them (just like we don't include build numbers of Windows in the Windows RIDs). Modify the host such that it will strip the minor version for `ol` when computing current RID. The rest of the system should not care in that case.

Reply:
> Why would we have to fix everything? 

AFAIK, they are using source-build. Fixing the host will probably be the largest part of the effort, but we will need to update a number of other places like build scripts too.

Reply:
I looked through all of .NET 7 source-build sources and after a quick pass, it looks like these files might also need fixups (it's possible I missed something, though):

```
runtime/eng/native/init-distro-rid.sh
runtime/src/installer/tests/scripts/linux-test/RuntimeInstallation.sh
runtime/src/installer/tests/scripts/linux-test/SdkInstallation.sh
runtime/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs
runtime/src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props
runtime/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json
runtime/src/libraries/Microsoft.NETCore.Platforms/src/runtime.compatibility.json
runtime/src/native/corehost/hostmisc/pal.unix.cpp
sdk/src/Cli/Microsoft.DotNet.Cli.Utils/RuntimeEnvironment.cs
sdk/src/Tests/Microsoft.NET.TestFramework/EnvironmentInfo.cs
fsharp/scripts/init-tools.sh
fsharp/scripts/dotnet-install.sh
deployment-tools/eng/native/init-distro-rid.sh
deployment-tools/src/clickonce/native/projects/setup.cmake
deployment-tools/src/installer/Directory.Build.props
diagnostics/eng/init-distro-rid.sh
arcade/src/Microsoft.DotNet.XUnitConsoleRunner/src/common/AssemblyResolution/Microsoft.DotNet.PlatformAbstractions/Native/PlatformApis.cs
```

Reply:
Also, there's a question about what to do in 3.1 and 6.0. Do we fix up host and other files there too? Otherwise, like discussed earlier in this issue, we would have to add minor versions in 6.0 and then add them here too for the sake of compatiblity.